src/locales/en/en.json

```diff
@@ -1191,21 +1191,10 @@
   "request": {
     "signtransaction": {
       "title": "Sign",
+      "label": "Cardano Ballot",
```

Comment:
This is not translatable - this label always comes from the name set by the dApp itself